src/lib/pages/proposal/whitelist/components/AssetBox.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, chakra, Flex, Image, Text } from "@chakra-ui/react";
+
+import { useNativeTokensInfo } from "lib/app-provider";
+import type { Option } from "lib/types";
+
+const AssetLabel = chakra(Text, {
+  baseStyle: {
+    top: 0,
+    left: 0,
+    mx: 3,
+    pt: 2,
+    mt: 2,
+    paddingInlineStart: 1,
+    paddingInlineEnd: 1,
+    position: "absolute",
+    fontWeight: "400",
+    lineHeight: "1.2",
+    pointerEvents: "none",
+    transformOrigin: "left top",
+    color: "text.dark",
+    fontSize: "16px",
+    transform: "scale(0.75) translateY(-29px)",
+    bg: "background.main",
+  },
+});
+
+export const AssetBox = ({ baseDenom }: { baseDenom: Option<string> }) => {
```

Comment:
Move this to component/proposal since it will be used in proposal store code too